cli/src/args.rs

```diff
@@ -19,14 +19,26 @@ pub const ARG_OUTPUT_FOLDER: &str = "output-folder";
 pub const ARG_FOLLOW_LINKS: &str = "follow-links";
 pub const ARG_TARGET_OS: &str = "target_os";
 
-#[cfg(feature = "go")]
-const AVAILABLE_LANGUAGES: [&str; 6] = ["kotlin", "scala", "swift", "typescript", "go", "python"];
+// #[cfg(all(feature = "go", feature = "python"))]
+// const AVAILABLE_LANGUAGES: [&str; 6] = ["kotlin", "scala", "swift", "typescript", "go", "python"];
+
+// #[cfg(all(feature = "go", not(feature = "python")))]
+// const AVAILABLE_LANGUAGES: [&str; 5] = ["kotlin", "scala", "swift", "typescript", "go"];
 
-#[cfg(not(feature = "go"))]
-const AVAILABLE_LANGUAGES: [&str; 5] = ["kotlin", "scala", "swift", "typescript", "python"];
+// #[cfg(all(feature = "python", not(feature = "go")))]
+// const AVAILABLE_LANGUAGES: [&str; 5] = ["kotlin", "scala", "swift", "typescript", "python"];
+
+// #[cfg(not(any(feature = "go", feature = "python")))]
+// const AVAILABLE_LANGUAGES: [&str; 4] = ["kotlin", "scala", "swift", "typescript"];
 
 /// Parse command line arguments.
 pub(crate) fn build_command() -> Command<'static> {
+    let languages: Vec<&str> = vec!["kotlin", "scala", "swift", "typescript"];
```

Comment:
This needs to be mutable. Otherwise, you'd end up with an error when trying to install with any of the features below.

Reply:
Right.. I tested it before committing thinking I had it working.. oops fixed